.github/workflows/android_alarm_manager_plus.yaml

```diff
@@ -23,15 +23,16 @@ jobs:
     timeout-minutes: 30
     steps:
       - name: "Checkout repository"
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: "Install Flutter"
         run: ./.github/workflows/scripts/install-flutter.sh stable
       - name: "Install Tools"
         run: ./.github/workflows/scripts/install-tools.sh
 
-      # Required for Android builds
-      - uses: actions/setup-java@v1
+        # Required for Android builds
+      - uses: actions/setup-java@v2
```

Comment:
I experimented with removing this step as suggested in one of other PRs (because of https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md#java saying that such runners already have Java 11), but couldn't build the project due to missing Java 11. Thus, returned the step again.